### ISSUE TITLE
[question] Get the logits

### ISSUE THREAD

**Issue by fede-bello** [dreamquark-ai/tabnet#557]
Hey! Really good implementation, I've been getting really good results.

I've look for this but I haven't found anything, is there a way to get the logits of the prediction? (the values before the last layer). I'm trying to use the model as a feature extractor for a dataset with a lot of features

**Comment by Optimox** [comment]
It's not directly feasible within the standard pipeline of the library, however if you define your tabnet network you can easily replace the last layer with an Identity layer. Or you can simply remove it from the source code.

**Comment by fede-bello** [comment]
I believe it could be a nice little feature to add a param or a method that does this
